src/odemis/driver/andorcam2.py

```diff
@@ -828,21 +828,18 @@ def __init__(self, name, role, device, emgains=None, shutter_times=None,
             # 0 => shutter always auto
             # > 0 => shutter auto if exp time + readout > period, otherwise opened
             # big value => shutter always opened
```

Comment:
This comment doesn't really make sense to me, is it meant something like this:
```suggestion
            # To control the shutter: select the maximum frequency, aka minimum
            # period for the shutter. If it the acquisition time is below the minimum period, the
            # shutter stays open all the time. So:
            # minimum period = 0 => shutter always auto
            # minimum period > 0 => shutter auto if exp time + readout > period, otherwise opened
            # big value => shutter always opened
```

Reply:
Indeed, that was quite unclear. I've reworded the text. Especially, I dropped the examples with "= 0, >0, big value", because it is actually not how it is used in reality, so it doesn't help much to understand this setting.